bldsys/cmake/template/gradle/app.build.gradle.in

```diff
@@ -2,13 +2,14 @@ plugins {
 	id 'com.android.application'
 }
 
-ext.vvl_version='1.3.275.0'
+ext.vvl_version='1.3.296.0'
 apply from: "./download_vvl.gradle"
 
 android {
 	compileSdk 33
-	buildToolsVersion "33.0.0"
-	ndkVersion "25.2.9519653"
+	buildToolsVersion '34.0.0'
```

Comment:
35.0.0 is available.  Also might not need to use buildToolsVersion, actually I think it's deprecated?

Reply:
See comment above.

Reply:
Merged your PR, so this should be addressed.